common/lib/common-utils/src/bufferBrowser.ts

```diff
@@ -29,6 +29,28 @@ export function Uint8ArrayToString(arr: Uint8Array, encoding?: string): string {
     }
 }
 
+/**
+ * Convert base64 or utf8 string to array buffer
+ */
+export const stringToBuffer = (input: string, encoding: string): ArrayBufferLike =>
+    IsoBuffer.from(input, encoding).buffer;
+
+/**
+ * Convert binary blob to string format
+ *
+ * @param blob - the binary blob
+ * @returns the blob in string format
+ */
+export const bufferToString = (blob: ArrayBufferLike): string => IsoBuffer.from(blob).toString("utf8");
```

Comment:
This is not consistent with above, in a sense that encoding is hard-coded here, but can be anything in stringToBuffer()
Either hard-code (or use default values) in both places, or make it explicit in both places